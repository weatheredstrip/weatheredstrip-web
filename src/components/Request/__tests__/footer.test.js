import React from 'react'
import { render, cleanup } from '@testing-library/react'
import Footer from '../footer.react'
import '@testing-library/jest-dom/extend-expect'
afterEach(cleanup)

it('footer renders properly', () => {
  const { container } = render(<Footer />)
  const footer = container.firstChild
  expect(footer).toMatchSnapshot()
})
