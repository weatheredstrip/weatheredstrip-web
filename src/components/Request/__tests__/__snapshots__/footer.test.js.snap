// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer renders properly 1`] = `
<div
  class="footer"
>
  <div
    class="footer-content"
  >
    <div
      class="footer-item"
    >
      Data from 
      <a
        href="https://flightplanning.navcanada.ca"
      >
        NAV CANADA
      </a>
      ‚Ñ†, 
      <a
        href="https://www.aviationweather.gov"
      >
        AWC
      </a>
      ‚Ñ† and
       
      <a
        href="https://notams.aim.faa.gov"
      >
        FAA
      </a>
      ‚Ñ†
    </div>
    <div
      class="footer-item"
    >
      <div>
        <a
          href="https://github.com/GregoryHamel/weatheredstrip/issues"
        >
          Report an Issue
        </a>
      </div>
      <div>
        ¬©
         
        <a
          href="http://www.greghamel.com"
        >
          Greg Hamel - 
          2020
        </a>
      </div>
    </div>
    <div
      class="footer-item"
    >
      Made in
       
      <span
        aria-label="Canada"
        role="img"
      >
        üçÅ
      </span>
       
      with
       
      <span
        aria-label="love"
        role="img"
      >
        ‚ù§Ô∏è
      </span>
    </div>
  </div>
</div>
`;
